resources/i18n/fr.json

```diff
@@ -274,7 +274,7 @@
             "not_found": "Page manquante",
             "show": "%{name} #%{id}",
             "empty": "Pas encore de %{name}.",
-            "invite": "Voulez-vous en créer un ?"
+            "invite": "Voulez-vous en créer ?"
```

Comment:
This is a tough one. My understanding is that this is a generic message that can be displayed in various situations. In the original message, "un" refers to the thing to create. "Un" is masculine, so it works well in use cases where the _thing_ is masculine.
I came across it while navigating the playlist page. Unfortunately, "playlist" is feminine in French, so it looks weird. The message _should_ definitely read "Voulez-vous en créer **une** ?"
![image](https://github.com/navidrome/navidrome/assets/7327649/1b3b2cdf-a32a-4d70-907c-25f978fc7d3a)

I think our best option is to completely get rid of this word. In my opinion, it works equally well as before. It may even work better, as the message now suggests the user can create _one or more_ thing if they want to.